src/Centreon/Application/ApiPlatform.php

```diff
@@ -28,27 +28,27 @@
 class ApiPlatform
 {
     /**
-     * @var float
+     * @var string
      */
     private $version;
 
     /**
      * Get the API version
      *
-     * @return float
+     * @return string
      */
-    public function getVersion(): float
+    public function getVersion(): string
```

Comment:
The is no more need to keep type convertion in "PlatformTopologyRegisterRepositoryAPI" class private function getToken line 93